HLTriggerOffline/B2G/python/b2gHLTValidationHarvest_cff.py

```diff
@@ -13,6 +13,26 @@
         resolution = cms.vstring(""),
         )
 
+b2gDoubleLeptonEleMuHLTValidationHarvest = cms.EDAnalyzer("DQMGenericClient",
```

Comment:
is this harvester added to any central harvesting sequence?
I don't see it in the b2gHLTriggerValidationHarvest one.